### ISSUE TITLE
Sourcemap for "/__uno.css" points to missing source files

### ISSUE THREAD

**Issue by leosin** [unocss/unocss#2113]
After upgrading nuxt to 3.1.1 yesterday, it prompts `Sourcemap for "/__uno.css" points to missing source files`. Is this a problem with nuxt or uncss?

![image](https://user-images.githubusercontent.com/3676670/214770424-aaafe83b-6bd6-4a31-b5b3-1caa7991851e.png)

**Comment by Amdeo** [comment]
I am same too

**Comment by notKamui** [comment]
I concur ; I don't get this issue on a regular vite+vue project, only with Nuxt (used with the module preset)

**Comment by Joaojack** [comment]
same here

**Comment by crazy6995** [comment]
same here +1

**Comment by so1ve** [comment]
Confirmed on Nuxt 3.1.1+. I'm using 3.1.2

**Comment by Gudsbarn** [comment]
WARN  Sourcemap for "/__uno.css" points to missing source files
The warning is still showing up in Nuxt 3.1.2 when using @unocss/nuxt and @unocss/reset
Maybe this issue could be moved to nuxt/nuxt?

**Comment by cuebit** [comment]
> WARN Sourcemap for "/__uno.css" points to missing source files
> The warning is still showing up in Nuxt 3.1.2 when using @unocss/nuxt and @unocss/reset
> Maybe this issue could be moved to nuxt/nuxt?

How confident are you that this is a Nuxt only issue and has no relation to unocss?

Perhaps open an issue on the Nuxt repo for further visibility but this should stay here as all roads lead here.

**Comment by AqueleHaru** [comment]
same here.

**Comment by reslear** [comment]
same

**Comment by Joaojack** [comment]
well i thought this was a response

**Comment by leosin** [comment]
```js
// nuxt.config.ts
  sourcemap: {
    server: true,
    client: false,
  },
```
This is the way I can think of so far, I don't know if it is the only way.

**Comment by Sandros94** [comment]
> ```js
> // nuxt.config.ts
>   sourcemap: {
>     server: true,
>     client: false,
>   },
> ```
> 
> This is the way I can think of so far, I don't know if it is the only way.

this doesn't work for me as of:
``` json
{
    "devDependencies": {
        "@unocss/nuxt": "^0.50.4",
        "nuxt": "^3.3.1",
    }
}

```

but now, when I run `npm ci`, I see:
```
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
```
but I don't know if this could be related...

**Comment by so1ve** [comment]
> 

I think this deprecation notice doesn't matter.

**Comment by joseehilton147** [comment]
new warning on nuxt 3.3.1, dont know if its nuxt fault or uno fault.

![image](https://user-images.githubusercontent.com/53986027/225608178-da063621-2214-4dba-a1e7-5e30c6e7ae16.png)

**Comment by joseehilton147** [comment]
@antfu didnt work the new update.

![image](https://user-images.githubusercontent.com/53986027/225618046-7ceaac42-cc49-40c9-9962-f5fc5e649d66.png)


![image](https://user-images.githubusercontent.com/53986027/225617973-c96de16f-be7a-4110-9891-3e60f45f3345.png)

**Comment by Sandros94** [comment]
Same here, v0.50.5 didn't do much, still getting:
```
 WARN  Sourcemap for "virtual:nuxt:LOCAL-PATH-TO-REPO/.nuxt/unocss.mjs" points to missing source files                                                                                           15:31:46  
 WARN  Sourcemap for "/�/__uno.css" points to missing source files  
```
P.S.: I'm happy to test anything.

**Comment by cuebit** [comment]
FYI, this issue is resolved in Vite v4.2.1. Can confirm this issue is resolved in Nuxt Edge and will be too in the next Nuxt stable release.
